starters/next12-react-query-tailwind/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "next12-react-query-tailwind",
+  "description": "NextJS, React Query, and TailwindCSS",
+  "keywords": [
+    "next",
```

Comment:
Do we need to create two keywords? 
one for next12 and another for next13?

Reply:
Should be nextjs. package.json keywords are for npm search and the versioning doesn't quite matter there.

Reply:
It also maps to technologies on the website so it has to match up with that and we don't want different next js coming up